docs/website/docs/dlt-ecosystem/verified-sources/rest_api.md

```diff
@@ -9,6 +9,54 @@ import Header from './_source-info-header.md';
 
 This is a generic dlt source you can use to extract data from any REST API. It uses [declarative configuration](#source-configuration) to define the API endpoints, their [relationships](#define-resource-relationships), how to handle [pagination](#pagination), and [authentication](#authentication).
 
+### Quick example
+
+Here's an example of how to configure the REST API source to load posts and related comments from a hypothetical blog API:
+
+```py
+import dlt
+from rest_api import rest_api_source
+
+source = rest_api_source({
+    "client": {
+        "base_url": "https://api.example.com/",
+        "auth": {
+            "token": dlt.secrets["your_api_token"],
+        },
+        "paginator": {
+            "type": "json_response",
+            "next_url_path": "paging.next",
+        },
+    },
+    "resources": [
+        "posts",
```

Comment:
one suggestion: maybe add a python comment here that explains that a simple entry of "posts" will hit https://api.example.com/posts and no further configuration for this endpoint is needed.